api/views/yangSearch/yangSearch.py

```diff
@@ -550,16 +550,22 @@ def get_modules_revision_organization(module_name: str, revision: t.Optional[str
     """
     try:
         if revision is None:
-            hits = app_config.es_manager.get_sorted_module_revisions(ESIndices.AUTOCOMPLETE, module_name)
+            hits = app_config.es_manager.get_sorted_module_revisions(ESIndices.YINDEX, module_name)
         else:
             module = {'name': module_name, 'revision': revision}
-            hits = app_config.es_manager.get_module_by_name_revision(ESIndices.AUTOCOMPLETE, module)
+            hits = app_config.es_manager.get_module_by_name_revision(ESIndices.YINDEX, module)
 
         organization = hits[0]['_source']['organization']
         revisions = []
         for hit in hits:
             hit = hit['_source']
-            revisions.append(hit['revision'])
+            revision = hit['revision']
+            revision_mat_level = {
+                'revision': revision,
+                'maturity_level': hit['rfc'] if 'rfc' in hit else False,
```

Comment:
Please either name this property `maturity_level` and return its value (like ratified, adopted ...) OR name this property `is_rfc` and then return True/False.